doc/DeveloperGuide.md

```diff
@@ -90,6 +90,31 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User send request to rename a tag
+2. The Addressbook validates and confirm the request
+3. User authorize the change by accepting this request from system
+4. Tag have been changed by the system
+
+**Extensions**
+
+1a. Tag cannot be found 
+> 1a-1. System will prompt a message notification
```

Comment:
The extensions here do not exactly follow the format. Please refer to the format above.